Index: src/storage/Storage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package storage;\r\n\r\npublic class Storage {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/storage/Storage.java b/src/storage/Storage.java
--- a/src/storage/Storage.java	(revision f6ef0e11d85b9b85ef42990c291a40c67a8f1866)
+++ b/src/storage/Storage.java	(date 1778051582979)
@@ -1,4 +1,38 @@
 package storage;
 
+import model.Destillering;
+import model.Fad;
+import model.Lager;
+
+import java.util.ArrayList;
+
 public class Storage {
+    private static ArrayList<Destillering> destilleringer;
+    private static ArrayList<Fad> fade;
+    private static ArrayList<Lager> lagre;
+
+
+    public static void addDestillering(Destillering destillering) {
+
+    }
+
+    public static void addFad(Fad fad) {
+
+    }
+
+    public static void addLager(Lager lager) {
+
+    }
+
+    public static ArrayList<Destillering> getDestilleringer() {
+        return null;
+    }
+
+    public static ArrayList<Fad> getFade() {
+        return null;
+    }
+
+    public static ArrayList<Lager> getLagre() {
+        return null;
+    }
 }
